<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DBuscar</class>
 <widget class="QDialog" name="DBuscar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>187</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Buscar: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leBuscar"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="chMayus">
         <property name="text">
          <string>Mayusculas/minusculas</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="chAtras">
         <property name="text">
          <string>CheckBox</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="btnBuscar">
       <property name="text">
        <string>Buscar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCerrar">
       <property name="text">
        <string>Cerrar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
